addons/web/static/src/js/views/form_upgrade_widgets.js

```diff
@@ -23,7 +23,7 @@ var AbstractFieldUpgrade = {
         this._super.apply(this, arguments);
         this.get_enterprise_label().after($("<span>", {
             text: "Enterprise",
-            'class': "label label-primary oe_inline"
+            'class': "label label-primary oe_inline oe_enterprise_label"
```

Comment:
@kujiu What is the usage of this class ? It's not used anywhere.

NB: new odoo classes are prefixed by `o_`. Before we used `oe_` for openerp.